config/src/loaders/toml.rs

```diff
@@ -234,18 +234,16 @@ enabled = false
         // Check that the tapi table does not need to explicitly set all the new "oppose_wip" fields
         // and they default to "false"
         let empty_config = super::from_str("[tapi]").unwrap();
-        let config_oppose_0016 = super::from_str(
+        let config_oppose_0017 = super::from_str(
             r"
 [tapi]
-oppose_wip0016 = true
+oppose_wip0017 = true
     ",
         )
         .unwrap();
 
         assert_eq!(empty_config.tapi, Tapi::default());
-        assert!(!empty_config.tapi.oppose_wip0014);
-        assert!(!empty_config.tapi.oppose_wip0016);
-        assert!(!config_oppose_0016.tapi.oppose_wip0014);
-        assert!(config_oppose_0016.tapi.oppose_wip0016);
+        assert!(!empty_config.tapi.oppose_wip0017);
+        assert!(config_oppose_0017.tapi.oppose_wip0017);
```

Comment:
This test was also used to check that unset fields default to false. Can you add something like that again?

        assert!(!empty_config.tapi.oppose_wip0018);
        assert!(!config_oppose_0017.tapi.oppose_wip0018);